### DISCUSSION TITLE
Feature request: Text editing application

### DISCUSSION THREAD

**Discussion by manu20247** [helloSystem/hello#567] [Ideas]
I would like to request a text editing application for the office, a text editing application dedicated to users who write a lot is essential.  I leave a model below to serve as a fork.  Thanks.

**Comment by manu20247** [comment]
![Screenshot_20240416_115228_Telegram](https://github.com/helloSystem/hello/assets/163211417/520793b7-d979-49ce-8305-a74fbf428618)

**Comment by probonopd** [comment]
Thanks for your suggestion @manu20247. If we ship a text editing app, it would most likely be modeled roughly after [TextEdit](https://en.wikipedia.org/wiki/TextEdit).

**Reply by manu20247** [reply]
oh my god, it's perfect.  very beautiful, minimalist, practical, visually elegant and light.  Thank you very much for your dedication.  Your work is excellent.  I can't wait to use it on Hello Desktop.
